### PR TITLE
Notifications

### PR DESCRIPTION
**Pull request by jakai7**
This PR implements a notification system with Redux and React-Toastify.

## Usage

```typescript
import {notify} from './redux/actions/notifications';

const notification = {
     text: "Hello World",
     color: "success"
};

dispatch(notify(notification));
```

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -28,123 +30,127 @@ import { BLOCK_EXPLORER_URL } from './config';
...
+                </a>
+              </h4>
+            </div>
+          ) : null}
+          {/* error */}
+          {transactions.pendingTransaction?.status ===
+          TransactionStatus.REJECTED ? (
+            <div>
+              <h3>Error</h3>
+              <h4>Transactoin rejected.</h4>
```
Comment (musnit) [inline src/App.tsx:132]: Transaction

Comment (musnit) [inline src/App.tsx:132]:
```suggestion
              <h4>Transaction rejected.</h4>
```

File: src/App.tsx [hunk 2]
```diff
@@ -28,123 +30,127 @@ import { BLOCK_EXPLORER_URL } from './config';
...
+          TransactionStatus.REJECTED ? (
+            <div>
+              <h3>Error</h3>
+              <h4>Transaction rejected.</h4>
+            </div>
+          ) : null}
+        </div>
+      </Modal>
+
+      {/* notifications */}
```
Comment (musnit) [inline src/App.tsx:138]: can remove this?

File: src/redux/reducers/index.ts [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
 import net from './net';
-import transactions from './transactions'
+import transactions from './transactions';
+import notifications from './notifications';
 
 export default combineReducers({
   net,
-  transactions
+  transactions,
+  notifications,
```
Comment (musnit) [inline src/redux/reducers/index.ts:9]: overall code looks good, but is this reducer actually even used anywhere? maybe we can just remove it if toastify is handling everything for us?

**Review by musnit — APPROVED** [review]

**Comment by musnit** [comment]
please merge in master + fix merge conflicts too

File: src/redux/reducers/notifications.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import {
```
Comment (musnit) [inline src/redux/reducers/notifications.ts:1]: is this file used/needed?

Comment (jakai7) [reply src/redux/reducers/notifications.ts:1]: No, we don't need it. I added this during my first attempt before I decided to use React-Toastify. I will remove this.

**Comment by musnit** [comment]
looks good - feel free to merge

**Pull request merged**